applications/sam2/start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# change the pythonpath
+export PYTHONPATH="/workspace/sam2"
```

Comment:
Suggest appending to the existing PYTHONPATH in case of users running on bare metal:
```suggestion
export PYTHONPATH="${PYTHONPATH}:/workspace/sam2"
```

If considering bare metal, you might also want to save the initial PYTHONPATH state and reset it at the end of the script to prevent lasting changes to the user's environment.

Reply:
added ${PYTHONPATH}: to both.